descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "item-storage",
-      "version": "8.1",
+      "version": "8.2",
```

Comment:
I don't think that we need to update neither "item-storage" nor "holdings-storage" nor "instances-storage" since nothing has updated in their schema.

Reply:
These changes are an artifact of initially basing this off on another PR. I must have forgotten **not** to commit these changes.